templates/tableau-cluster-linux.template

```diff
@@ -0,0 +1,1627 @@
+                            "\t\tmethod: 'PUT',",
+                            "\t\theaders: {",
+                            "\t\t\t'content-type': '',",
+                            "\t\t\t'content-length': responseBody.length",
+                            "\t\t}",
+                            "\t};",
+                            "",
+                            "\tconsole.log('SENDING RESPONSE...\\n');",
+                            "",
+                            "\tvar request = https.request(options, (response) => {",
+                            "\t\tconsole.log('STATUS: ' + response.statusCode);",
+                            "\t\tconsole.log('HEADERS: ' + JSON.stringify(response.headers));",
+                            "\t\tc.done(); // Tell AWS Lambda function execution is done",
+                            "\t});",
+                            "",
+                            "\trequest.on('error', (err) => {",
+                            "\t\tconsole.log('sendResponse Error:' + err);",
+                            "\t\tc.done(); // Tell AWS Lambda function execution is done",
+                            "\t});",
+                            "",
+                            "\t// write data to request body",
+                            "\trequest.write(responseBody);",
+                            "\trequest.end();",
+                            "}"
+                      ]]
+                  }
+                },
+                "Handler":"index.handler",
+                "Role":{
+                    "Fn::GetAtt":[
+                        "LambdaExecutionRole",
+                        "Arn"
+                    ]
+                },
+                "Runtime":"nodejs4.3",
+                "Timeout":"30"
+            }
+        },
+        "LambdaExecutionRole":{
+            "Type":"AWS::IAM::Role",
+            "Properties":{
+                "AssumeRolePolicyDocument":{
+                    "Version":"2012-10-17",
+                    "Statement":[
+                        {
+                            "Effect":"Allow",
+                            "Principal":{
+                                "Service":[
+                                    "lambda.amazonaws.com"
+                                ]
+                            },
+                            "Action":[
+                                "sts:AssumeRole"
+                            ]
+                        }
+                    ]
+                },
+                "Path":"/",
+                "Policies":[
+                    {
+                        "PolicyName":"root",
+                        "PolicyDocument":{
+                            "Version":"2012-10-17",
+                            "Statement":[
+                                {
+                                    "Effect":"Allow",
+                                    "Action":[
+                                        "logs:CreateLogGroup",
```

Comment:
Please use the managed service role policy for Lambda basic execution to get the logging permissions. Keep this policy just for the describe VPC permissions.

Here's a similar example: https://github.com/aws-quickstart/quickstart-git2s3/blob/master/templates/git2s3.template#L267-L269